uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/WorkPatternCalendar.php

```diff
@@ -83,6 +83,11 @@ public function get() {
    * method won't include that pattern unless the contact also has a contract
    * during that date.
    *
+   * For cases where there are time lapses between the Work Pattern Periods, These
+   * lapses are filled using the default Work Pattern. Also lapses between the start
+   * of the Absence period and the first Work pattern period, between the last Work Pattern
+   * period and the Absence period end date are filled using the default Work Pattern.
+   *
```

Comment:
I think the whole docblock description is still inaccurate. On the second paragraph, it says that dates that don't have a matching contract won't be included in the response, which is not true anymore.